vyper/typing.py

```diff
@@ -11,6 +11,7 @@
 OutputFormats = Sequence[str]
 OutputDict = Dict[ContractPath, OutputFormats]
 StorageLayout = Dict
+StorageLayoutForContracts = Dict[ContractPath, StorageLayout]
```

Comment:
I don't think this type is necessary since it is only used in one place and its purpose is already clear at the use site